### PR TITLE
Add `Locked` to `RunnerDetails` struct

### PR DESCRIPTION
**Pull request by rsrchboy**
It's available in the API, but not exposed until now.

### CODE REVIEW

File: runners.go [hunk 1]
```diff
@@ -77,6 +77,7 @@ type RunnerDetails struct {
 		Name   string `json:"name"`
 		WebURL string `json:"web_url"`
 	} `json:"groups"`
+	Locked         bool `json:"locked"`
```
Comment (svanharmelen) [inline runners.go]: These fields are (more or less) ordered according to the docs, so in that case this field should be above `AccessLevel`. And could you also add the field to the create and update option structs to make support for the field complete? Thanks!

Comment (rsrchboy) [reply runners.go]: I've moved it up; AFAICT the update and create structs already have `Locked`, though I might be missing something.

Comment (svanharmelen) [reply runners.go]: Your absolutely right! Thanks for fixing the PR!

**Comment by svanharmelen** [comment]
LGTM! Thanks @rsrchboy!

**Pull request merged**